lib/light_serializer/hashed_object.rb

```diff
@@ -48,7 +46,7 @@ def values_from_nested_resource(attribute, object, result)
     def hashed_nested_resource(value, nested_serializer)
       return hashed_entity(value, nested_serializer) unless can_be_enumerated?(value)
 
-      value.each_with_object(nested_serializer).map(&method(:hashed_entity))
+      value.each_with_object(nested_serializer).map { |entity, serializer| hashed_entity(entity, serializer) }
```

Comment:
https://github.com/JuanitoFatas/fast-ruby#normal-way-to-apply-method-vs-method-code